lang/python/python/files/python-package-openssl.mk

```diff
@@ -8,7 +8,7 @@
 define Package/python-openssl
 $(call Package/python/Default)
   TITLE:=Python $(PYTHON_VERSION) SSL module
-  DEPENDS:=+python-light +libopenssl
+  DEPENDS:=+python-light +libopenssl +ca-bundle
```

Comment:
i am little confused by the existence of both `ca-bundle` and `ca-certificates`
but i guess this is fine;